tests/BrandingClientTest.php

```diff
@@ -173,21 +180,24 @@ public function cachingTimesDataProvider()
         return [
             [
                 [],
-                ['Date' => 'Thu, 13 Oct 2016 16:10:30 GMT', 'Expires' => 'Thu, 13 Oct 2016 16:11:30 GMT'],
+                ['Date'    => 'Thu, 13 Oct 2016 16:10:30 GMT',
```

Comment:
Is PHPCS ok with this way of formatting the array?

These tests haven't changed so this ends up being diff noise. I note that you haven't changed the equivalent of this test in the OrbitClient test. This should either be done in both or neither.